kroxylicious-filters/pom.xml

```diff
@@ -16,15 +16,16 @@
         <version>0.4.0-SNAPSHOT</version>
     </parent>
 
-    <artifactId>kroxylicious-additional-filters</artifactId>
+    <groupId>io.kroxylicious.filter</groupId>
+    <artifactId>kroxylicious-filter-parent</artifactId>
 
-    <name>Additional Filters</name>
-    <description>Optional filters provided by the kroxylicious project</description>
+    <name>Filter implementations</name>
```

Comment:
The intent with calling these additional filters was to emphasise that they are optional and not required for the proxy to function. 

At one stage we had filters (IIRC `ApiVersionsFilter`) that were required to be in the filter chain and thus for the proxy to operate. While we have done away with mandatory filters I think the hint these are not required for the proxy operation is a useful one.

Reply:
Does the presence of the word `additional` here really help all that much, when there is (or will be) plenty of documentation about how to write your own, and how to configure the proxy with filter implementation? The person who configures which filter jars are present at runtime might not even interact with maven coordinates.

Reply:
It might not help people configuring the runtime but it will help people working on the project!